hdrass/CCW+Rotator_integration_testing/Rotator_interlock_test.ipynb

```diff
@@ -0,0 +1,1248 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "Reactivate the CCW"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "CCWstate = mtm.evt_cameraCableWrapFollowing.get()\n",
+    "print(\"CCWstate:\",CCWstate.enabled)\n",
+    "await mtm.cmd_enableCameraCableWrapFollowing.set_start(timeout=10.)\n",
+    "await CCWstate = await mtm.evt_cameraCableWrapFollowing.next(timeout=10.)\n",
+    "print(\"CCWstate:\",CCWstate.enabled)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "await salobj.set_summary_state(rot, salobj.State.ENABLED)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "await mtm.cmd_enterControl.start(timeout=30.)\n",
+    "await mtm.cmd_start.start(timeout=30.)\n",
+    "await mtm.cmd_enable.start(timeout=30.)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Set Low Velocity and Move Through Negative Interlock\n",
+    "This will move the rotator indepently from the CCW and will therefore activate the negative limit switch located at the bulkhead plate between the Rotator and the CCW"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#await rot.cmd_configureVelocity.set_start(vlimit=1.5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Disable the CCW"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "CCWstate = mtm.evt_cameraCableWrapFollowing.get()\n",
+    "print(\"CCWstate:\",CCWstate.enabled)\n",
+    "await mtm.cmd_disableCameraCableWrapFollowing.set_start(timeout=10.)\n",
+    "await CCWstate = mtm.evt_cameraCableWrapFollowing.next(timeout=10.)\n",
+    "print(\"CCWstate:\",CCWstate.enabled)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This will make the rotator fault"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "await asyncio.sleep(1.)\n",
+    "\n",
+    "print(\"Move to +10 deg position\")\n",
+    "\n",
+    "await rot.cmd_move.set_start(position=+10,timeout=30.)\n",
+    "\n",
+    "print(\"Reset the Rotator and CCW\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Bring the Rotator and CCW back to enabled state"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "await mtm.cmd_clearerror.start(timeout=30)"
```

Comment:
Some issue: clearError and it fails because it is not implemented. I suggest a global search/replace. Also I suggest writing a function that restores normal state, so you can reduce code duplication.

Reply:
All `clearError` commands are taken out.